distribution/docs/sender/application.properties

```diff
@@ -37,6 +37,18 @@ db-event.retry.initial.delay=120000
 
 # Number of events to sync in parallel, defaults to 10
 events.sync.parallel.size=
+
+# The interval in milliseconds before the first sender sync messages fetch by the active mq publisher route when the application starts.
+active-mq-publisher.initial.delay=120000
```

Comment:
2 minutes is a long time for initial delay

Reply:
Wyclif. 
In the first version, I set this delay to 15 seconds. Later on, in [this comment](https://github.com/FriendsInGlobalHealth/openmrs-eip/pull/11#discussion_r869834783), you suggested to change it to two minutes, to give time to other routes with more priority...

Now I'm confused seeing you saying that the two minutes that you said to be used is a long time!